addons/interaction/RscTitles.hpp

```diff
@@ -67,7 +67,7 @@ class RscACE_SelectAnItem {
             text = "";
         };
         class itemList: RscListBox {
-            onMouseButtonDblClick = "_this call ACE_Interaction_fnc_onSelectMenuDblClick";
+            onMouseButtonDblClick = QUOTE(_this call DFUNC(onSelectMenuDblClick));
```

Comment:
all these calls in this file don't exist

Reply:
parent `RscACE_SelectAnItem` isn't used anywhere in ace
might be a remnant from agm